components/stacks-network/src/chains_coordinator.rs

```diff
@@ -679,14 +679,7 @@ async fn handle_bitcoin_mining(
     devnet_event_tx: &Sender<DevnetEvent>,
 ) {
     let stop_miner = Arc::new(AtomicBool::new(false));
-    loop {
-        let command = match mining_command_rx.recv() {
-            Ok(cmd) => cmd,
-            Err(_e) => {
-                // TODO(lgalabru): cascade termination
-                break;
-            }
-        };
+    while let Ok(command) = mining_command_rx.recv() {
```

Comment:
I think I'd keep the initial version and add a log in the `Err` code path (and keep that todo)